airbyte-webapp/src/packages/cloud/locales/en.json

```diff
@@ -7,6 +7,8 @@
   "login.resendEmail": "Didn’t receive the email? Send it again",
   "login.yourEmail": "Your work email*",
   "login.yourEmail.placeholder": "christophersmith@gmail.com",
+  "login.yourEmail.notFound": "User not found",
+  "login.unknownError": "An unknown error has occurred",
```

Comment:
@andyjih any comment on this wording too?  `notFound` = Firebase can't find a user with that email.  `unknownError` = any other error is returned from the call.

Reply:
@tealjulia lgtm!